src/features/rewards/donate/style.ts

```diff
@@ -103,14 +123,30 @@ export const StyledSend = styled<{}, 'div'>('div')`
   cursor: pointer;
 `
 
-export const StyledSendButton = styled<{}, 'button'>('button')`
+export const StyledSendButton = styled<StyleProps, 'button'>('button')`
   display: block;
   border: none;
-  font-size: 13px;
+  font-size: ${p => p.monthly ? 12 : 13}px;
   font-weight: 600;
   text-transform: uppercase;
   background: var(--donate-send-bg);
   cursor: pointer;
+  ${(p) => {
```

Comment:
Could we create new style that would extend `StyledSendButton` and use that for monthly? This way it will be a lot cleaner

Reply:
done